utils/zerovec/src/lib.rs

```diff
@@ -231,9 +233,9 @@ mod yoke_impls;
 mod zerofrom_impls;
 
 pub use crate::error::ZeroVecError;
-pub use crate::map::map::ZeroMap;
 #[cfg(feature = "hashmap")]
-pub use crate::map::ZeroHashMapStatic;
+pub use crate::hashmap::ZeroHashMapStatic;
```

Comment:
thought: in the long run we need to figure out where everything lives. I would like to eventually have a separate zeromap crate with all of the different map types, though ZeroMapKV and the original ZeroMap might still live here.

I think there's a good chance to build map types that don't need the KV magic and can work based off of _containers_ rather than using the gettable type, which is definitely nice and magical but unfortunately causes a lot of complexity.